src/connection.h

```diff
@@ -40,7 +40,7 @@
 #include "ae.h"
 
 #define CONN_INFO_LEN 32
-#define CONN_ADDR_STR_LEN 128 /* Similar to INET6_ADDRSTRLEN, hoping to handle other protocols. */
+#define CONN_ADDR_STR_LEN 128 /* Similar to NET_ADDR_STR_LEN, hoping to handle other protocols. */
```

Comment:
now the line 47, NET_ADDR_STR_LEN is not used anymore, we can delete the define if needed. I'm keeping it for now in case we need it in the future.

Reply:
I think we should delete it. There are too many very similar defines and hard to know which one to use. If we keep it, we will forget it and keep it forever.